packages/compass-e2e-tests/helpers/commands/shell-eval.ts

```diff
@@ -7,10 +7,12 @@ async function getOutputText(browser: CompassBrowser): Promise<string[]> {
   });
 }
 
+let screenshotNumber = 0;
+
 export async function shellEval(
   browser: CompassBrowser,
   connectionName: string,
-  str: string,
+  input: string | string[],
```

Comment:
shellEval() now supports multiple lines. We used it consecutively in some tests and the problem is that before, we just showed and hid it so it kept its state, but now we open a new one and close it so it loses state. So tests that would `use database` broke because database reset to `test` every time.

Now we can give it multiple lines.

Ideally we'd probably not be using the shell like this for fixture data anyway, but that's probably a larger change.